tests/MongoDB.Driver.Tests/UpdateDefinitionBuilderTests.cs

```diff
@@ -239,18 +239,16 @@ public void Inc_Typed()
         }
 
         [Fact]
-        public void Indexed_Typed()
+        public void Incorrect_index_should_throw_expected_exception_with_set()
```

Comment:
I know that this file has a different naming style, but I guess this name describes the test idea more fully

Reply:
That's fine.